Apps/PackageTest/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "PackageTest",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@babylonjs/core": "^4.2.0-alpha.21",
+    "@babylonjs/react-native": "file:../Playground/node_modules/@babylonjs/react-native/babylonjs-react-native-0.0.1.tgz",
```

Comment:
This is the important part of this test app. All you need to do is `npm pack` in the `Apps/Playground/node_modules/@babylonjs/react-native` directory, then follow the normal steps in "initializing" the app (e.g. `npm install`, `pod install`, etc.).